doc/tutorials/validation.md

```diff
@@ -120,9 +120,9 @@ kdb setmeta spec/tutorial/spec/test hello world
 kdb set /tutorial/spec/test value
 #> Using name user/tutorial/spec/test
 #> Create a new key user/tutorial/spec/test with string "value"
-kdb lsmeta spec/tutorial/spec/test
+kdb lsmeta spec/tutorial/spec/test | grep -v 'internal/ini'
```

Comment:
The regex is too generic, it should start with an `^`.

I am also thinking about how to handle this completely without grep, see #1695